src/plugins/ncr/actions/nodesRecieved.js

```diff
@@ -0,0 +1,6 @@
+import { actionTypes } from '../constants';
```

Comment:
@nanetteCodes @efjacobson action creator should be imperative... e.g. loadNodes or receiveNodes and the action type is past tense (NODES_RECEIVED)